### DISCUSSION TITLE
validation and parameter passing at startup

### DISCUSSION THREAD

**Discussion by MrZmcH** [eclipse-theia/theia#13187] [Q&A]
How to do some validation according to the token in the cookie when starting theia? And Using` yarn start --myParam==‘’`, how do you get myParam parameters from inside theia

**Comment by JonasHelming** [comment]
see https://github.com/eclipse-theia/theia/pull/13155

**Reply by MrZmcH** [reply]
@JonasHelming thank you，This does not seem to help me, how can I call a validation method every time I open theia and refresh the page to achieve real-time validation

**Reply by JonasHelming** [reply]
Could you describe the use case more in detail? Real time validation is typically not something you would refresh the page for. If there is a UI that offers validation, you would handle the validation and real time validation inside the logic of this UI without affecting Theia.

**Reply by MrZmcH** [reply]
#9528 ，In fact, this is probably the problem I encountered, but there seems to be no good solution

**Reply by JonasHelming** [reply]
So you want to implement a log in? My answer was more targeted at the second question on how to get a parameter.
